cookbooks/provision/recipes/centos_provision_with_disks.rb

```diff
@@ -0,0 +1,37 @@
+require 'chef/provisioning'
+require 'chef/provisioning/vsphere_driver'
+
+#
+# This is the main way to connect to vSphere
+#
+with_vsphere_driver host: '172.16.20.2',
+  insecure: true,
+  user:     'administrator@vsphere.local',
+  password: 'Good4bye!'
+
+#
+# These are the machine_options that you need to declare
+#
+with_machine_options :bootstrap_options => {
+                               num_cpus: 2,
+                               memory_mb: 4096,
+                               datacenter: 'Datacenter',
+                               resource_pool: 'Cluster',
+                               template_name: 'centos7-tmpl',
+                               :ssh => {
+                                 :user => 'root',
+                                 :password => 'admini',
+                                 :paranoid => false,
+                               },
+                               bootstrap_ipv4: true,
+
+                               additional_disk_size_gb: [10, 20],
+                             }
```

Comment:
added the line:
`                               datastore: 'vsanDatastore',` here. 

I got it to pass but didn't see the disk's added. Could be broken code though.